inc/class-wprus-settings.php

```diff
@@ -565,9 +567,25 @@ public function get_site( $url, $action = false, $direction = false ) {
 
 			foreach ( $sites as $site ) {
 
-				if ( trailingslashit( $url ) === trailingslashit( $site['url'] ) ) {
+				/** 
+				 * note that in some cases, both the sending and master sites might use the same protocol ( for example https ) 
```

Comment:
@froger-me please note that this change was required in order for the plugin to work for me, perhaps this was a configuration issue and while I understand this might reduce the security this checks is meant to apply, it might also overcome some silent failures.